site/docs/api.md

```diff
@@ -99,6 +99,8 @@ Available operations to update a table are:
 
 ### Transactions
 
+`Table` also enables to commit multiple table operations at once.
```

Comment:
I think we should either remove this empty section or expand it a bit beyond what you have here.

Transactions are used to commit multiple table changes in a single atomic operation. A transaction is used to create individual operations using factory methods, like `newAppend`, just like working with a `Table`. But operations created by a transaction are committed as a group when `commitTransaction` is called.

```java
Transaction t = table.newTrasaction();

// commit operations to the transaction
t.newDelete().deleteFromRowFilter(filter).commit();
t.newAppend().appendFile(data).commit();

// commit all the changes to the table
t.commitTransaction();
```

Reply:
thanks for the suggestion. updated accordingly.